packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -12,6 +12,10 @@ import * as React from "react";
 
 import * as Classes from "../../common/classes";
 
+// amount in pixels that the content div width changes when truncated vs when
+// not truncated. Note: could be modified by styles
+const CONTENT_DIV_WIDTH_DELTA = 25;
```

Comment:
Well ideally we don't need to hardcode such things, a refactor is needed in the future